app/views/appropriate_bodies/participants/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <span class="govuk-caption-xl">Appropriate body</span>
```

Comment:
I'm not sure we need this anymore?

Reply:
I was thinking about this. There isn't any way to know you're in the AB section. We have a similar caption for DP. I'm not sure if we should remove this.